### PR TITLE
Focus comment box when entering by "comments" button from home feed

### LINKED ISSUE #17883: Make comment box in focus when entering by clicking "xxx comments" from home feed
# Current behavior & problem
While browsing posts on the feed-page, when the user clicks on the **Add comment** / **3 Comments**, it redirects users to the post-page comment area. 

At that time comment box is not focused, so the user has to click that comment box to start writing the comments. 

# New behavior
Idea is to make that comment box focused when you click **Add comment** / **3 Comments** on feed-page. 
This small interaction will improve the comment writing experience.

I believe we should also include what @aitchiss suggested in the [comment](https://github.com/forem/forem/issues/17883#issuecomment-1155023624) below:

> When we do this, can we please change the story cards (that appear on the various feeds) `aria-label` to make sure the link is accessible as something like "Add a comment to post - ${post_name}", to avoid any confusion for screen reader users 🙂

Image

### PR DESCRIPTION
**Pull request by jhonatan-kmt**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
On the feed-page, when the user clicks on the add comment, it redirects users to the post-page comment area.

## Related Tickets & Documents

- Related Issue #17883
- Closes #17883

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: the tests were already implemented.
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: app/javascript/articles/components/CommentsCount.jsx [hunk 1]
```diff
@@ -26,6 +26,7 @@ export const CommentsCount = ({ count, articlePath, articleTitle }) => {
         icon={commentsSVG}
         tagName="a"
         aria-label={commentsAriaLabelText}
+        tooltip={`Add a comment to post - ${articleTitle}`}
```
Comment (aitchiss) [inline app/javascript/articles/components/CommentsCount.jsx:29]:
```suggestion
        aria-label={`Add a comment to post - ${articleTitle}`}
```

I don't think it's part of the requirements to introduce a tooltip, but we do want the accessible name to reflect the action that's going to happen on click, so let's just switch these out 🙂

**Comment by jhonatan-kmt** [comment]
Hi @aitchiss, I have added the E2E tests, I hope you can check them.

**Review by aitchiss — COMMENTED** [review]
This is looking awesome and thank you for adding that test.

I failed to notice last time though that we need to add your aria-label changes a few more places because we also create feed story cards from:

- `app/assets/javascripts/utilities/buildArticleHTML.js` (used for paged in articles as you scroll down the home feed)
- `app/views/articles/_single_story.html.erb` (used in places like the user's profile, and the logged out feed)

Sorry I didn't pick up on that before! If you can update those aria-labels too then I think we're good to go 🚀

**Review by aitchiss — APPROVED** [review]
Love it! Thank you for making those small changes ❤️

**Review by aitchiss — APPROVED** [review]

**Review by joshpuetz — APPROVED** [review]
Test locally and works! Thanks much!

**Pull request merged**